ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/ForkChoiceUpdateData.java

```diff
@@ -208,6 +208,18 @@ public ForkChoiceState getForkChoiceState() {
     return forkChoiceState;
   }
 
+  private boolean doesNotRequireSend(final UInt64 currentTimestamp) {
```

Comment:
nit: I'd probably call this `isSentRecently` mostly because the negative hurts my brain, but not strong on it so feel free to ignore.